tests/foreman/api/test_template_combination.py

```diff
@@ -77,3 +78,19 @@ def test_positive_get_combination(self):
         self.assertEqual(self.template.id, combination.config_template.id)
         self.assertEqual(self.env.id, combination.environment.id)
         self.assertEqual(self.hostgroup.id, combination.hostgroup.id)
+        combination.delete()
```

Comment:
Is combination deletion really mandatory (i mean for example not deleted combination will affect other entities or lead to serious performance degradation or affect another test somehow)?
This ways tests are not isolated and in case delete functionality is broken, test for `read()` will fail too, so in case there's no real need to delete combination here other that clean things up i'd better remove this line.